invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -365,20 +367,169 @@ def view(**kwargs):
     return view
 
 
-def add_signposting(f):
-    """Add signposting link to view's response headers."""
+def _get_header(rel, value, link_type=None):
+    header = f'<{value}> ; rel="{rel}"'
+    if link_type:
+        header += f' ; type="{link_type}"'
+    return header
+
+
+def _get_signposting_cite_as(record):
+    """Release self url points to RDM record.
+
+    It points to DataCite URL if the integration is enabled, otherwise it points to the HTML URL.
+    """
+    doi_url = record["links"].get("doi")
+    html_url = record["links"]["self_html"]
+    return _get_header("cite-as", doi_url or html_url)
+
+
+def _get_signposting_types(record):
+    resource_type = record["metadata"]["resource_type"]
+    props = get_vocabulary_props(
+        "resourcetypes",
+        [
+            "props.schema.org",
+        ],
+        resource_type["id"],
+    )
+    url_schema_org = props.get("schema.org")
+    return [
+        _get_header("type", url_schema_org),
+        _get_header("type", "https://schema.org/AboutPage"),
+    ]
+
+
+def _get_signposting_authors(record):
+    authors = []
+    # Limit authors to the first 10.
+    for creator in islice(record["metadata"]["creators"], 0, 10):
+        for identifier in creator["person_or_org"].get("identifiers", []):
+            if identifier["scheme"] == "orcid":
+                authors.append(
+                    _get_header(
+                        "author", "https://orcid.org/" + identifier["identifier"]
+                    )
+                )
+    return authors
```

Comment:
Lars suggested that we might choose to not include authors at all since the list might be long and the full list can be found in the linkset.

Reply:
Maybe we could apply some sensible limit? E.g. if less than 50 authors, include, otherwise don't include at all and basically have people rely on the explicit authors linkset?